### PR TITLE
refactor: import wallet

### PR DESCRIPTION
**Pull request by guanbinrui**
## Description

Closes MF-0000

## Type of change

- [ ] Documentation
- [x] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Review by Jack-Works — DISMISSED** [review]

File: packages/app/src/plugin-host/enable.ts [hunk 1]
```diff
@@ -15,14 +15,15 @@ import { ChainId } from '@masknet/web3-shared-evm'
...
+import * as services from '../plugin-worker/service.js'
+// import { PluginWorker } from './rpc.js'
 
 // #region Setup storage
 const inMemoryStorage = createKVStorageHost(
     {
         beforeAutoSync: Promise.resolve(),
-        getValue: worker.memoryRead,
-        setValue: worker.memoryWrite,
+        getValue: services.memoryRead,
```
Comment (guanbinrui) [inline packages/app/src/plugin-host/enable.ts:27]: CC @Jack-Works Need fix this.

File: packages/app/src/components/SidebarForDesktop.tsx [hunk 2]
```diff
@@ -74,10 +75,15 @@ export function SidebarForDesktop(props: SidebarForDesktopProps) {
...
-                                className="flex items-center gap-x-4 px-6 py-3 text-sm font-semibold leading-6 text-white hover:bg-gray-800">
+                                className="flex items-center gap-x-4 px-6 py-3 text-sm font-semibold leading-6 text-white hover:bg-gray-800"
+                                onClick={() => {
+                                    console.log('DEBUG: open')
+
+                                    SelectProviderModal.open()
+                                }}>
                                 <img
                                     className="h-8 w-8 rounded-full bg-gray-800"
-                                    src="https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80"
```
Comment (Jack-Works) [inline packages/app/src/components/SidebarForDesktop.tsx:80]: who is that in the original photo...

File: packages/app/src/main/index.tsx [hunk 3]
```diff
@@ -17,7 +17,6 @@ export function DecryptUI() {
     if (!message) return <Typography>Decrypting...</Typography>
     return (
         <Suspense>
-            {/* Do not add React context here. Add it in ./plugin-render */}
```
Comment (Jack-Works) [inline packages/app/src/main/index.tsx:20]: remove the comment might bring mistake in the future.

File: packages/app/src/components/SidebarForDesktop.tsx (hunk 2 above)
Comment (guanbinrui) [reply packages/app/src/components/SidebarForDesktop.tsx:80]: Just a template code.

File: packages/shared-base/src/Site/index.ts [hunk 4]
```diff
@@ -4,6 +4,7 @@ import { ExtensionSite, EnhanceableSite } from './types.js'
 
 const matchEnhanceableSiteHost: Record<EnhanceableSite, RegExp> = {
     [EnhanceableSite.Localhost]: /localhost/i,
+    [EnhanceableSite.App]: /app\.mask\.io|netlify\.app|localhost/i,
```
Comment (Jack-Works) [inline packages/shared-base/src/Site/index.ts]: maybe remove netlify.app, it includes so many untrusted sites.

and please ensure matches with the hostname, not the path. here is some example of evil URLs that can match this expression:

- app.mask.io.evil.com
- anything.netlify.app
- localhost.evil.com
- evil.com/localhost
- app.mask.ioevil.com

File: packages/polyfills/types/flag.d.ts [hunk 5]
```diff
@@ -8,6 +8,7 @@ declare module NodeJS {
          * Debug flags
          */
         readonly BUILD_DATE: string
+        readonly APP: boolean
```
Comment (Jack-Works) [inline packages/polyfills/types/flag.d.ts:11]: Please use feature detect. do not add new flags

**Pull request merged**